App/Redux/CameraRollRedux.ts

```diff
@@ -21,24 +21,6 @@ const actions = {
   }),
   untrackPhotos: createAction('UNTRACK_PHOTOS', resolve => {
     return (ids: string[]) => resolve({ ids })
-  }),
-  imagePickerSuccess: createAction('IMAGE_PICKER_SUCCESS', resolve => {
-    return (threadId: string, image: TextileTypes.SharedImage) => resolve({ threadId, image })
-  }),
-  addComment: createAction('ADD_IMAGE_PICKER_COMMENT', resolve => {
-    return (threadId: string, image: TextileTypes.SharedImage, caption: string) => resolve({ threadId, image, caption })
-  }),
-  localPinSuccess: createAction('LOCAL_PIN_SUCCESS', resolve => {
-    return (threadId: string, image: TextileTypes.SharedImage, addResult: TextileTypes.AddResult) => resolve({ threadId, image, addResult })
-  }),
-  remotePinStarted: createAction('REMOTE_PIN_STARTED', resolve => {
-    return (threadId: string, image: TextileTypes.SharedImage) => resolve({ threadId, image })
-  }),
-  imagePinError: createAction('IMAGE_PICKER_PIN_ERROR', resolve => {
-    return (threadId: string, image: TextileTypes.SharedImage) => resolve({ threadId, image })
-  }),
-  cancelShare: createAction('CANCEL_IMAGE_PICKER_SHARE', resolve => {
-    return (threadId: string, image: TextileTypes.SharedImage) => resolve({ threadId, image })
```

Comment:
It was confusing to have image sharing redux spread around, so it's all in `UIRedux` now

Reply:
perfect